lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1315,7 +1295,7 @@ def deploy_vm(self):
         self.configure_cpu_and_memory(vm_obj=vm_obj, vm_creation=True)
         self.configure_disks(vm_obj=vm_obj)
         self.configure_network(vm_obj=vm_obj)
-        if self.should_deploy_from_template() and len(self.params['customize']) > 0:
+        if vm_obj is None or self.should_deploy_from_template():
```

Comment:
Ok we can use sysprep with from scratch VM. But if we run customization in all cases why this condition ?
I think it should be, if we set customization params : if len(self.params['customize']) > 0: